deploy/charts/csi-driver/templates/daemonset.yaml

```diff
@@ -19,10 +19,6 @@ spec:
         - name: node-driver-registrar
           image: "{{ .Values.nodeDriverRegistrarImage.repository }}:{{ .Values.nodeDriverRegistrarImage.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          lifecycle:
-            preStop:
-              exec:
-                command: ["/bin/sh", "-c", "rm -rf /registration/cert-manager-csi-driver /registration/cert-manager-csi-driver-reg.sock"]
```

Comment:
This command removes both plugin (?) and socket whereas the fix in https://github.com/kubernetes-csi/node-driver-registrar/pull/61 seems to only remove the socket- do we think that might be an issue?

I do see that the https://github.com/kubernetes-csi/node-driver-registrar/pull/61 removes this command from the example deployment, so assume that this is considered no longer needed.

Reply:
That it my understanding as well.